code/game/machinery/doors/windowdoor.dm

```diff
@@ -295,6 +299,33 @@
 		if("deny")
 			flick("[src.base_state]deny", src)
 
+/obj/machinery/door/window/check_access_ntnet(datum/netdata/data)
+	return !requiresID() || ..()
+
+/obj/machinery/door/window/ntnet_recieve(datum/netdata/data)
+	// Check if the window is powered and can accept control packets.
+	if(!hasPower())
+		return
+
+	// Check packet access level.
+	if(!check_access_ntnet(data))
+		return
+
+	// Handle recieved packet.
+	var/command = lowertext(data.plaintext_data)
+	var/command_value = lowertext(data.plaintext_data_secondary)
+	switch(command)
+		if("open")

```

Comment:
should be toggle not open

Reply:
actually is this the same as airlocks?

Reply:
Yeah I copied it from airlocks and removed the bolt and emergency commands.

Reply:
Should the airlock command be changed to "toggle" as well? Either way, I feel like it should be consistent across both door types.

Reply:
Also removed AI check as it doesn't have attack_ai AFAIK.

Reply:
Leave it as "open"

Reply:
airlock have 3 commands:"bolt", "open", and "emergency".It goes to primary text field.Open/bolt is selfdescriptive.Emergency disables access check.
These commands have 3 possible values."on","off","toggle".It goes to secondary text.

also, windoors should could to be bolted,btw.

Reply:
You certainly can stuck windoor open. Ingame.
But after looking at code, I can't say if it's bug or intended feature. Look at close proc.

Reply:
I can see that you can emag a door. I can also see that both the open and close proc support a `forced` parameter, but cannot find a call to it except in `/obj/machinery/door/airlock/receive_signal(datum/signal/signal)`.

Reply:
The `locked` and `emergency` variable is not modified in windowdoor.dm and the parent door.dm, only airlock.dm.

Reply:
Well, I can lock windoor open in game. And can't find, where it is in code.